electroncash_gui/qt/address_list.py

```diff
@@ -358,37 +358,6 @@ def doCopy(txt):
 
         run_hook('address_list_context_menu_setup', self, menu, addrs)
 
-        # Add Cash Accounts section at the end, if relevant
-        if not multi_select:
```

Comment:
I'm curious what this was meant for?

Reply:
This is used by BCH Cash Accounts. If you right click on the address it had these Cash Account menu items.